### DISCUSSION TITLE
libero project creation

### DISCUSSION THREAD

**Discussion by rroulleau** [polarfire-soc/polarfire-soc-documentation#67] [Reference designs, FPGA and development kits] [answered]
When running the script to create the project it is looking for F: drive location. 
See the script execution report.

Error: Failure when executing Tcl script. [ Line 48: Internal Error:
Assertion Failed in "F:/release/sn/capture_sjts68_joy_jen_sn/afi/prg/pfsoc_mss/src/pa5socmssApp.cpp", line 192.

[microsemi_ScriptExecutionReport.txt](https://github.com/polarfire-soc/icicle-kit-reference-design/files/5267431/microsemi_ScriptExecutionReport.txt)

**Comment by hughbreslin** [comment, marked as answer]
Hey @rroulleau does the path to the script have any spaces in it? I can reproduce a similar failure if I run using a path with a space.

**Comment by rroulleau** [comment]
Correct, there was space in the path of the project! Thanks a lot!
